Dockerfile

```diff
@@ -2,6 +2,8 @@
 FROM openjdk:8-jre-alpine
 LABEL maintainer="Martin DSouza <martin@talkapex.com>"
 
+RUN apk --no-cache add --update curl
```

Comment:
Can we add this to the list of other ‘RUN’ commands?

Also was ‘curl’ not included in base alpine?

Reply:
change it already. 

It seems like base alpine does not include curl...